android/src/main/java/com/chargebee/android/models/PurchaseResult.kt

```diff
@@ -0,0 +1,12 @@
+package com.chargebee.android.models
+
+import com.chargebee.android.network.ReceiptDetail
+import java.io.Serializable
+
+data class PurchaseResult (
+  val subscriptionId: String?,
```

Comment:
Any reason to keep `subscriptionId` & `planId` optional?

Reply:
In the [apidoc](https://apidocs.chargebee.com/docs/api/in_app_subscriptions?prod_cat_ver=2), these fields are marked as optional and hence I have followed it here. 
I have raised it internally for verifying why they are optional.

Reply:
Confirmed that subscriptionId & planId aren't optional and that the change would be done in our Apidocs accordingly. Will update this model.